lib/config.js

```diff
@@ -407,8 +407,14 @@ Config.prototype.getConfig = function(filePath) {
     // Step 8: Merge in command line globals
     config = util.mergeConfigs(config, { globals: this.globals });
 
+    // Step 9: Merge in command line plugins
+    if (this.options.plugins) {
```

Comment:
Ah yes, I see now, I can do something like https://github.com/eslint/eslint/blob/master/tests/lib/config.js#L433-L456.  I'll add it in tonight.